tool/travis.sh

```diff
@@ -39,6 +39,12 @@ elif [ "$CHECK_BOT" = true ] ; then
   # Check plugin-referenced urls for liveness.
   dart tool/grind.dart check-urls
 
+elif [ "$UNIT_TEST_BOT" = true ] ; then
+  # Run unit tests without failing the build.
+  set +e
```

Comment:
can we remove this `set +e` and `exit 0` given the `ignoreFailures true` in the gradle file?